app/Controllers/LibriController.php

```
@@ -102,6 +103,19 @@ private function getCoversUrlPath(): string
         return '/uploads/copertine';
     }
 
+    private function syncSeriesMetadataFromBookForm(mysqli $db, int $bookId, array $fields, array $data): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)